tests/feature_extraction/test_extraction.py

```diff
@@ -201,3 +202,6 @@ def test_extract_features(self):
         self.assertTrue(np.all(extracted_features.b__abs_energy == np.array([36619, 35483])))
         self.assertTrue(np.all(extracted_features.b__mean == np.array([37.85, 34.75])))
         self.assertTrue(np.all(extracted_features.b__median == np.array([39.5, 28.0])))
+
+if __name__ == '__main__':
```

Comment:
why is this needed for your feature? I would rather go with `pytest test_extraction.py` instead of calling the files by itself, because we set additional settings for pytest only (and not for unittest), which could in principle lead to strange things like the test is working here but not there...